A change in Qt 5.11_beta3 leads to errors

Arch GNU/Linux

```
CMake Error at src/libqtdbusmock/CMakeLists.txt:194 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```
```
CMake Error at tests/CMakeLists.txt:21 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```
According to [the official Qt documentation](https://doc.qt.io/qt-5/cmake-manual.html) `qt5_use_modules` macro is obsolete 

Same as https://github.com/ubports/ubuntu-download-manager/issues/4

--- a/src/libqtdbusmock/CMakeLists.txt
+++ b/src/libqtdbusmock/CMakeLists.txt
@@ -191,11 +191,11 @@ add_library(
 	${QTDBUSMOCK_SRC}
 )
 
-qt5_use_modules(
-	qtdbusmock
-	Core
-	DBus
-	Test
+target_link_libraries(
+    qtdbusmock
+    Qt5::Core
+    Qt5::DBus
+    Qt5::Test
 )
 
 target_link_libraries(

--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -18,11 +18,11 @@ add_executable(
 	${UNIT_TESTS_SRC}
 )
 
-qt5_use_modules(
-	unit-tests
-	Core
-	DBus
-	Test
+target_link_libraries(
+    unit-tests
+    Qt5::Core
+    Qt5::DBus
+    Qt5::Test
 )
 
 target_link_libraries(

